packages/fluentui/docs/src/prototypes/meetingOptions/components/MSTeamsMeetingOptions.tsx

```diff
@@ -18,7 +18,12 @@ export default () => {
         <TransparentDivider size={40} />
         <Header as="h3" content="Meeting Options" />
         <TransparentDivider />
-        <Grid columns="1fr 1fr">
+        <Grid
+          styles={{
+            gridTemplateColumns: '1fr 1fr',
+            msGridColumns: '1fr 1fr',
+          }}
+        >
```

Comment:
```suggestion
        <Grid columns={2}>
```

I think that it will be the same result